From af5c59b98dedbd732043a1f9155a94fb96158383 Mon Sep 17 00:00:00 2001
From: Aikar <aikar@aikar.co>
Date: Fri, 1 Apr 2016 00:02:47 -0400
Subject: [PATCH] Add FastUtil to Bukkit

Doesn't expose to plugins, just allows Paper-API to use it for optimization

diff --git a/pom.xml b/pom.xml
index a613cf963..f8f12595b 100644
--- a/pom.xml
+++ b/pom.xml
@@ -50,6 +50,12 @@
     </pluginRepositories>
 
     <dependencies>
+        <dependency>
+            <groupId>it.unimi.dsi</groupId>
+            <artifactId>fastutil</artifactId>
+            <version>8.2.2</version>
+            <scope>provided</scope>
+        </dependency>
         <dependency>
             <groupId>commons-lang</groupId>
             <artifactId>commons-lang</artifactId>
-- 
2.20.1

